### ISSUE TITLE
Physical keyboard uses English US layout if magnification (accessibility) is enabled (Astro Slide only)

### ISSUE THREAD

**Issue by wocard2** [shymega/planet-devices#47]
If Settings -> Accessibility -> Magnification is On for the Astro Slide 5G Transformer, the physical keyboard ("Gemini Keyboard") uses a minimal English US layout (which unfortunately lacks most mappings involving the Fn key),  no matter what layout is configured in Planet Keyboard. The Cosmo Communicator did not have that problem.

**Comment by TauPan** [comment]
This also gets triggered by enabling "reading mode" (Lesemodus in German, translation guessed by me).
